fetch.bs

```diff
@@ -2215,6 +2288,86 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<p>When <dfn>recording connnection timing info</dfn> given a <var>connection</var>, perform the
+following steps:
```

Comment:
As this cannot be implemented as a step-by-step algorithm I think this would work better as a section that describes the requirements. What do you think?

Reply:
I think an unordered list of requirements would make more sense. I'm also not sure what some of the conditionals add here. E.g., the cache conditional doesn't seem to change that start is before lookup and end is after. That there's a cache involved can just be an implementation detail.